main.go

```diff
@@ -27,6 +27,8 @@ func init() {
 	flag.BoolVar(&config.EnableHTTP, "enable-http", true, "run server in HTTP mode as well")
 	flag.BoolVar(&config.EnableMultiThreading, "enable-multithreading", false, "run server in multithreading mode")
 	flag.IntVar(&config.HTTPPort, "http-port", 8082, "HTTP port for the dice server")
+	flag.BoolVar(&config.EnableWebsocket, "enable-websocket", true, "run server in Websocket mode as well")
```

Comment:
Where are we using this flag?

Reply:
No we are not using this flag. I kept it to be consistent with http. Can you please help me understand where was http flag used earlier?

Reply:
@lucifercr07 - please suggest what to do here.

Reply:
@psrvere we can implement flag as part of another PR.

Reply:
Sure, removed flag from this PR.